<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>department_widget</class>
 <widget class="QWidget" name="department_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lDepartmentCode">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Department code:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="leDepartmentCode"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lDepartmentName">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Department name:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lEDepartmentName"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lChairPerson">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Chair Person:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lEChairPerson"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lOfficeLocation">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Office location:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QTextEdit" name="tEOfficeLocation"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="lBudget">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Budget:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lEBudget"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="lEstablishedDate">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Established date:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDateEdit" name="dEEstablishedDate"/>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QCheckBox" name="chDepActive">
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
